src/assets/scripts/client/ui/SettingsController.js

```diff
@@ -184,13 +184,13 @@ export default class SettingsController {
      * @private
      */
     _buildOptionSelectTemplate(optionData, selectedOption) {
-        // the `selectedOption` coming in to this method will always be a string (due to existing api) but
-        // could contain valid numbers. here we test for valid number and build `parsedSelectedOption` accordingly.
-        const parsedSelectedOption = !_isNaN(parseFloat(selectedOption)) ?
-            parseFloat(selectedOption) :
-            selectedOption;
+        // // the `selectedOption` coming in to this method will always be a string (due to existing api) but
```

Comment:
if this is no longer needed then lets remove it

Reply:
done